new/components/basic/form/ErrorCode.tsx

```diff
@@ -0,0 +1,12 @@
+interface ErrorCodeProps {
+    errorList: string[],
+}
+const ErrorCode = ({ errorList }: ErrorCodeProps) => {
+    return (
+        <h5>
+            {errorList.map((error, index) => <div key={index}>{`*${error}!`}</div>)}
+        </h5>
```

Comment:
should use those Basic wrapper, or react-native `<Text>` if absent

Reply:
Done